asset/snippets/updateSavedQuery.js

```diff
@@ -40,15 +39,31 @@ async function main(fullQueryName, description) {
     };
 
     // Handle the operation using the promise pattern.
-    const result = await client.updateSavedQuery(request);
+    const [query] = await client.updateSavedQuery(request);
     // Do things with with the response.
-    console.log(util.inspect(result, {depth: null}));
+    console.log("Query name:", query.name);
+    console.log("Query description:", query.description);
+    console.log("Created time:", query.createTime);
+    console.log("Updated time:", query.lastUpdateTime);
+    console.log("Query type:", query.content.queryContent);
+    console.log("Query content:", JSON.stringify(query.content, null, 4));
     // [END asset_quickstart_update_feed]
+    return query;
```

Comment:
Per the [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/#result), we should avoid `return` statements.

Instead, let's [verify the correct output was logged](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/bd5e6a0f830290833997a3fa18a28fe5b44fef5d/functions/pubsub/subscribe/test/index.test.js#L44-L45) using something like the [`stubConsole` function shown here](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/bd5e6a0f830290833997a3fa18a28fe5b44fef5d/functions/pubsub/subscribe/test/index.test.js#L23-L34).

_Please apply this comment throughout your PR - thanks! 🙂_